src/Util/Log/JUnit.php

```diff
@@ -229,6 +229,7 @@ public function startTestSuite(TestSuite $suite)
     {
         $testSuite = $this->document->createElement('testsuite');
         $testSuite->setAttribute('name', $suite->getName());
+        $testSuite->setAttribute('package', $suite->getName());
```

Comment:
This does not make sense to me. `$suite->getName()` is probably not the package name.